docs/src/components/mdx/code-block-theme.ts

```diff
@@ -0,0 +1,199 @@
+import { PrismTheme } from 'prism-react-renderer'
+
+// import { colorsTextIcon } from '@pluralsight/ps-design-system-core'
+
```

Comment:
@sweeting can you help me develop the proper theme colors.

i think we can use this tool to create the theme https://k88hudson.github.io/syntax-highlighting-theme-generator/www/

the current docs only have a dark theme...not sure if we want to keep that as is or add light theme as well